src/foam/nanos/ruler/RulerDAO.js

```diff
@@ -106,8 +107,42 @@ foam.CLASS({
       of: 'foam.mlang.predicate.Predicate',
       name: 'removeAfter',
       javaFactory: `return AND(
+  EQ(Rule.ASYNC, false),
   EQ(Rule.OPERATION, Operation.REMOVE),
   EQ(Rule.AFTER, true)
+);`
```

Comment:
again here think the body of javaFactory just needs an indent

Reply:
same as above, indentation for the generated java code.